qa-test

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
+
+# Shellcheck
+find . -path ./.git -prune -o -type f -exec grep -Eq '^#!(.*/|.*env +)(sh|bash|ksh)' {} \; -print |
```

Comment:
suggestion for more performance with big files:
find . -type f -exec bash -c "head -n1 {} | grep -Eq '^#!(.*/|.*env +)(sh|bash|ksh)'" \; -print